theatre/studio/src/panels/DetailPanel/prop-editors/DetailDeterminePropEditor.tsx

```diff
@@ -0,0 +1,60 @@
+import type {
+  PropTypeConfig,
+  PropTypeConfig_AllSimples,
+} from '@theatre/core/propTypes'
+import type SheetObject from '@theatre/core/sheetObjects/SheetObject'
+import type {Pointer} from '@theatre/dataverse'
+import React from 'react'
+import DetailCompoundPropEditor from './DetailDeterminePropEditor/DetailCompoundPropEditor'
+import DetailSimplePropEditor from './DetailDeterminePropEditor/DetailSimplePropEditor'
+import {getPropTypeByPointer} from '@theatre/studio/panels/DetailPanel/prop-editors/utils/getPropTypeByPointer'
+import {simplePropEditorByPropType} from '@theatre/studio/panels/DetailPanel/prop-editors/simple-editors/simplePropEditorByPropType'
+import type {PropConfigForType} from './utils/PropConfigForType'
+import type {ISimplePropEditorReactProps} from './simple-editors/ISimplePropEditorReactProps'
+
+const DetailDeterminePropEditor: React.VFC<
```

Comment:
Would read better as: `DeterminePropEditorForDetailPanel`.

Also, wish this was a rename rather than a new file. Losing history here, and hard to see what changed.

Reply:
Other related Detail components are prefixed with Detail, should those become "...ForDetailPanel"?